File: choose_mutants.py
import pandas as pd

data = pd.read_csv("868_TCR_to_pMHC_contacts_clean.txt", sep="\t")
print data["Donor_Annotation "]
cdrs = data[data["Donor_Annotation "].isin(["CDR3a", "CDR3b"])]

print cdrs

File: choose_mutants.R
setwd("~/PycharmProjects/STACEI")

data <- read.csv("868_TCR_to_pMHC_contacts_clean.txt", sep="\t", row.names = NULL)
data <- data[data$Donor_ResNum %in% c("CDR3a", "CDR3b"),]
data <- data[data$Acceptor_Chain_Letter == "peptide",]

possible <- data %>%
  group_by(Donor_ResNum, Donor_Annotation, Donor_Chain) %>%
  tally()